spec/sources/git_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'spec_helper'
-require 'danger/scm_source/git'
+require 'danger/scm_source/git_repo'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.